ci-cd-platform-deployment/README.md

```diff
@@ -0,0 +1,396 @@
+# CI/CD Platform Deployment
+
+As people start new projects, they usually waste precious time deploying and configuring a CI/CD pipeline from scratch.
+At Zenika Labs, our goal is to deliver proofs of concept or minimum viable products as efficiently as possible, without
+compromising on quality.
+
+This guide describes how to setup, in probably less than an hour, the *feature branch* workflow we use everyday to 
+build, test and deploy our applications. It uses Google's managed Kubernetes (GKE) as well as CircleCI, Traefik and 
+others Google Cloud services.
+
+**What is Kubernetes?**
+> Kubernetes, also known as K8s, is an open-source system for automating deployment, scaling, and management of 
+containerized applications. In the past years, Kubernetes has become the de-facto industrial standard to deploy containers on-premise or in the
+cloud.
+ 
+If you have never used Kubernetes before, this guide is probably not for you. Start by 
+[learning a bit about Kubernetes](https://kubernetes.io/docs/concepts/overview/) first.
+
+**What is CircleCI?**
+> CircleCI is a cloud-native continuous integration and continuous delivery (CI/CD) platform. It integrates with GitHub
+and Bitbucket and runs a configured pipeline on each commit. Think Jenkins multibranch pipelines on steroids, in the 
+cloud, and fully managed for you.
+ 
+If you have never used CircleCI before, welcome aboard and enjoy the [free plan](https://circleci.com/pricing/)!
+
+**What is Traefik?**
+> Traefik is a dynamic, modern, open-source Edge Router that automatically inspects your infrastructure to discover 
+services and how to route traffic to them. Traefik is natively compliant with every major cluster technology, such as 
+Kubernetes, Docker, Docker Swarm, AWS, Mesos, Marathon... and even bare metal! Used as an ingress provider in 
+Kubernetes, it is probably a drop-in replacement for the default provider and brings awesome features such as automated
```

Comment:
Je ne comprends pas le fait d'être un remplacement du "default provider" 🤔  De base il n'y a pas d'Ingress Controller dans Kubernetes. Dans GKE il  y a bien un Ingress Controller par défaut par contre.

Reply:
You're right c'était pas clair. J'ai reformulé.